include/lsst/daf/base/DateTime.h

```diff
@@ -57,19 +59,27 @@ namespace base {
 
 class DateTime {
 public:
-    enum Timescale { TAI, UTC, TT };
-    enum DateSystem { JD, MJD, EPOCH };
+    enum DateSystem { JD=0, MJD, EPOCH };
+    enum Timescale { TAI=5, UTC, TT };  // use values that do not overlap DateSystem
+                                        // to avoid confusing one for the other in Python
```

Comment:
The problem you had the other day was that from the Python side MJD and UTC had the same integer value so the C++ code just worked by fluke.